cmd/clusterctl/cmd/root.go

```diff
@@ -30,6 +30,21 @@ var RootCmd = &cobra.Command{
 	Use:   "clusterctl",
 	Short: "cluster management",
 	Long:  `Simple kubernetes cluster management`,
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		// This is a temporary hack to enable proper logging until upstream dependencies
+		// are migrated to fully utilize klog instead of glog.
+		klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
+		klog.InitFlags(klogFlags)
+
+		// Sync the glog and klog flags.
+		cmd.Flags().VisitAll(func(f1 *pflag.Flag) {
```

Comment:
I'm actually wondering if we should put this as a helper function into klog ... it seems broadly applicable!